### ISSUE TITLE
Implement homepage design from mockups

### ISSUE THREAD

**Issue by eporter23** [emory-libraries/dlp-selfdeposit#680]

**Comment by eporter23** [comment]
Yes! We think a lot of this will be front-end, but some of it will require backend, so I can also write up new tickets for those.

**Comment by maxdmayhew** [comment]
https://github.com/emory-libraries/dlp-selfdeposit/pull/730

**Comment by maxdmayhew** [comment]
https://github.com/emory-libraries/dlp-selfdeposit/pull/738

<img width="611" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMEtWQ0E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--d44c734045f01a13a3bff3b7fa90d60360ecede9/image.png" alt="image.png" />
****

**Comment by maxdmayhew** [comment]
https://github.com/emory-libraries/dlp-selfdeposit/pull/741

mobile and statistics finished out
